app/main/forms/direct_award_forms.py

```diff
@@ -39,3 +38,13 @@ class DidYouAwardAContractForm(FlaskForm):
             (STILL_ASSESSING, 'We are still assessing services')
         ]
     )
+
+class WhyDidYouNotAwardForm(FlaskForm):
+    why_did_you_not_award_the_contract = RadioField(
```

Comment:
If we just called this variable `reason` then in quite a few places I think the code would be more readable?